pkgs/development/python-modules/pyqtwebengine/default.nix

```diff
@@ -21,20 +22,21 @@ in buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace "[tool.sip.project]" "[tool.sip.project]''\nsip-include-dirs = [\"${pyqt5}/${python.sitePackages}/PyQt5/bindings\"]"
+      --replace "[tool.sip.project]" "[tool.sip.builder]''\njobs = $NIX_BUILD_CORES''\n[tool.sip.project]''\nverbose = true''\nsip-include-dirs = [\"${pyqt5}/${python.sitePackages}/PyQt5/bindings\"]"
   '';
 
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [
     pkg-config
     qmake
-    sip
+    buildPackages.python3Packages.sip
```

Comment:
Let's see what the consequences of passing spliced as self in makeScopeWithSplicing arehttps://github.com/NixOS/nixpkgs/pull/265136

Reply:
I don't understand this comment.

Aren't we trying to move away from using unspliced packages in dependency attributes like `nativeBuildInputs`?

Reply:
The change you have here now is necessary because of https://github.com/NixOS/nixpkgs/blob/ad83c5cf4b7b05b134e9698ff3f30069829d7fa6/pkgs/top-level/python-packages.nix#L10730-L10732 `self` doesn't contain spliced packages

Reply:
:rage: :rage: :rage:  I hate splicing. :rage: :rage: :rage: